recipes/awherepy/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "awherepy" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b7a1c5a09432f6c1870499bbde8f7ee33396794a9ee1df5196fcc0665bb971a3
+
+build:
+  number: 0
+  skip: True  # [py<38]
```

Comment:
Please remove this line and add `noarch: python`.  You can say `python >=3.8` below

Reply:
Copy that. Should I specify `python >= 3.8` for both the `host` and `run` sections?